f600233d1b154d507b782f84604bcb12/update/sys_script_include_da81226197072110cf3cfd9fe153afee.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.MarketResearchReport</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description/>
+        <name>MarketResearchReport</name>
+        <script><![CDATA[var MarketResearchReport = Class.create();
+MarketResearchReport.prototype = {
+
+    errUtil: new ErrorHandler(),
+    acquisitionPackage: null,
+    janda: null,
+    fairOpportunity: new GlideRecord("x_g_dis_atat_fair_opportunity"),
+    DATA_INCOMPLETE: 4001,
+    UNEXPECTED_ERROR: 5001,
+
+    initialize: function(acqPackage) {
+        this.acquisitionPackage = acqPackage;
+        this.janda = new JustificationAndApproval(acqPackage);
+        this.constructError = this.janda.constructError;
+        this.explicitNull = this.janda.explicitNull;
+
+        this.requirementsChecklist = new RequirementsChecklist(acqPackage);
+
+        // Look up Fair Opportunity record
+        this.fairOpportunity.get(this.acquisitionPackage.fair_opportunity);
+        if (!this.fairOpportunity.sys_id) {
+            throw this.constructError(this.DATA_INCOMPLETE, "No Fair Opportunity record found");
+        }
+
+    },
+
+    explicitNull: function(value) {
```

Comment:
Should be able to re-use the J&A version of this (see `this.explicitNull = this.janda.explicitNull;` from `initialize`).

Reply:
good idea

Reply:
Removed in https://github.com/dod-ccpo/atat-snow/pull/392/commits/3d7f5576f56fb68d072c873b92739a0f778e2a51